pinot-core/src/main/java/org/apache/pinot/core/query/scheduler/QueryScheduler.java

```diff
@@ -315,7 +313,7 @@ private boolean forceLog(long schedulerWaitMs, long numDocsScanned) {
    */
   protected ListenableFuture<byte[]> immediateErrorResponse(ServerQueryRequest queryRequest,
       ProcessingException error) {
-    DataTable result = new DataTableImplV2();
+    DataTable result = new DataTableImplV3();
```

Comment:
Fix this as per https://github.com/apache/incubator-pinot/pull/6710/files#r604379681

Reply:
done